userservice/pom.xml

```diff
@@ -102,23 +96,15 @@
 			<artifactId>hibernate-core</artifactId>
 			<version>6.2.2.Final</version>
 		</dependency>
-
-		<dependency>
-			<groupId>io.springfox</groupId>
-			<artifactId>springfox-swagger2</artifactId>
-			<version>3.0.0</version>
-		</dependency>
-
```

Comment:
Can you add the Swagger and Swagger UI dependency to this file?

Reply:
sure